awslimits/manage.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sendgrid
+from flask_script import Manager
+
+from server import app
+from support import get_limits_for_alert, alert_email_body
+
+manager = Manager(app)
+
+
+@manager.command
+def send_alerts():
```

Comment:
I have been thinking about the best way to avoid spamming people. I can definitely keep track of things people have been notified. I was originally thinking of comparing the alert against open AWS tickets but hadn't come up a clean way to reliably parse out the limit type from our tickets yet. I suppose the second option assumes that we will choose to open a ticket for every alert we receive, which is likely but may not always be the case. 

At the very least I can set the alert to only be sent if the same alert hasn't been sent in the last 3 (or however many we think is appropriate) days. Or did you have something else in mind?